src/cron/generateInvites.ts

```diff
@@ -0,0 +1,55 @@
+import fastq from 'fastq';
+import { Cron } from './cron';
+import { Feature, FeatureValue, Invite } from '../entity';
+import { generateUUID } from '../ids';
+
+const cron: Cron = {
+  name: 'generate-invites',
+  handler: async (con, logger) => {
+    const featuresQuery = con
+      .createQueryBuilder()
+      .from(Feature, 'f')
+      .where(
+        'f."createdAt" < now() - interval \'7 days\' AND f."value" = :value',
+        { value: FeatureValue.Allow },
```

Comment:
We also need to make sure we look for search feature, not any feature right? This is specifically for search

Reply:
The invitation system is generic, but this logic is specifically to search. Every feature will have different logic, I assume

Reply:
That's a good point, we generalised it indeed.
In that case we should probably rename to `generate-search-invites` as well.